_clients/data-prepper/index.md

```diff
@@ -18,32 +18,32 @@ Data Prepper is compromised of **Pipelines** that collect and filter data based
 
 - One [source](#source)
 - One [buffer](#buffer)
-- One or more [processors](#processor)
-- One of more [sinks](#sink)
+- (Optional) One or more[processors](#processor)
+- (Optional) One or more[sinks](#sink)
```

Comment:
Sinks are not optional. Only the processors are optional.

Reply:
Are buffer's optional?